force-app/main/default/lwc/RD2_RecurringDonation/RD2_RecurringDonation.js

```diff
@@ -0,0 +1,292 @@
+
+    lastDonationDate = '';
+
+    labels = {
+      commonAmount,
+      RDCL_Frequency,
+      lblStatus,
+      firstDonation,
+      mostRecentDonation,
+      nextDonation,
+      lastModified,
+      RD2_ViewMoreDetails,
+      RD2_ViewLessDetails
+    }
+
+    data = [{
+        "id": 1,
+        "firstDonation": "7/4/2022",
+        "donor": "Saba",
+        "amount": 69,
+        "paymentMethod": "Credit Card, *1111Exp. 11/2023",
+        "frequency": "Monthly on the 22nd",
+        "status": "Active",
+        "mostRecentDonation": "12/10/2023",
+        "lastModified": "4/01/2023",
+        "nextDonation": "<div class='slds-truncate dv-dynamic-width'>Paid Donation: 1 of 5.<br/> Next Donation: 1/12/2022.</div>"
+      },
+      {
+        "id": 2,
+        "firstDonation": "4/10/2022",
+        "donor": "Siward",
+        "amount": 43,
+        "paymentMethod": "ACH, *1111",
+        "frequency": "Monthly on the 22nd",
+        "status": "Active",
+        "mostRecentDonation": "12/10/2023",
+        "lastModified": "4/01/2023",
+        "nextDonation": "<div class='slds-truncate dv-dynamic-width'>Paid Donation: 3 of 6.<br/> Next Donation: 12/20/2022.</div>"
+      },
+      {
+        "id": 3,
+        "firstDonation": "3/31/2022",
+        "donor": "Page",
+        "amount": 43,
+        "paymentMethod": "Credit Card, *1111Exp. 11/2023",
+        "frequency": "Monthly on the 22nd",
+        "status": "Active",
+        "mostRecentDonation": "12/10/2023",
+        "lastModified": "4/01/2023",
+        "nextDonation": "<div class='slds-truncate dv-dynamic-width'>Paid Donation: 2 of 5.<br/> Next Donation: 2/22/2022.</div>"
+      },
+      {
+        "id": 4,
+        "firstDonation": "6/25/2022",
+        "donor": "Eric",
+        "amount": 91,
+        "paymentMethod": "Check",
+        "frequency": "Monthly on the 22nd",
+        "status": "Active",
+        "mostRecentDonation": "12/10/2023",
+        "lastModified": "4/01/2023",
+        "nextDonation": "<div class='slds-truncate dv-dynamic-width'>Paid Donation: 2 of 10.<br/> Next Donation: 11/11/2022.</div>"
+      },
+      {
+        "id": 5,
+        "firstDonation": "5/18/2022",
+        "donor": "Merrill",
+        "amount": 10,
+        "paymentMethod": "Credit Card",
+        "frequency": "Monthly on the 22nd",
+        "status": "Active",
+        "mostRecentDonation": "12/10/2023",
+        "lastModified": "4/01/2023",
+        "nextDonation": "<div class='slds-truncate dv-dynamic-width'>Paid Donation: 6 of 8.<br/> Next Donation: 12/22/2022.</div>"
+    }];
+
+    actions = [
+      { label: 'View', name: 'view' },
+      { label: 'Delete', name: 'delete' }
+    ];
+
+    columns = [];
+    
+    @wire(getObjectInfo, { objectApiName: RECURRING_DONATION })
+    oppInfo({ data, error }) {
+        if (data){
+          this.paymentMethod = data.fields.PaymentMethod__c.label;
+        }
+    }
+
+    connectedCallback() {
+      if(!this.isMobile){
```

Comment:
Maybe this could be moved into the @wire method and the "connectedCallback" method be deleted? Just a suggestion

Reply:
In this case, the validation must be done in the connected callback since we would be ensuring that it is done at the moment the component is inserted into the DOM, however, in the wire it would evaluate at the moment the call to the service ends.